test/SILOptimizer/devirt_access.swift

```diff
@@ -4,18 +4,6 @@
 // RUN: %target-swift-frontend -O -primary-file %s %S/Inputs/devirt_access_helper.swift -I %t -emit-sil -sil-inline-threshold 1000 -sil-verify-all | %FileCheck -check-prefix=WHOLE-MODULE %s
 // RUN: %target-swift-frontend -O -primary-file %s %S/Inputs/devirt_access_helper.swift -I %t -emit-sil -sil-inline-threshold 1000 -sil-verify-all | %FileCheck -check-prefix=PRIMARY-FILE %s
 
-import devirt_access_other_module
```

Comment:
The "explicit non-resilient" mode would just disable the new diagnostics. I'd be OK with making them conditional if we really had to, but I think basically this test was just exercising that the SIL optimizer doesn't blow up in a case that no longer type checks.